crates/polars-ops/src/chunked_array/list/count.rs

```diff
@@ -18,8 +23,12 @@ fn count_bits_set_by_offsets(values: &Bitmap, offset: &[i64]) -> Vec<IdxSize> {
 
             let len = (end - current_offset) as usize;
 
-            let set_ones = len - count_zeros(bits, bitmap_offset + current_offset as usize, len);
-            set_ones as IdxSize
+            // Fast path where all bits are set.
```

Comment:
This branch should be outside of the loop.

Reply:
I tried to avoid (almost) duplicating the logic of the loop.